src/main/java/com/icthh/xm/ms/entity/web/rest/XmEntityResource.java

```diff
@@ -338,13 +350,15 @@ public XmEntityResource(
     @GetMapping("/xm-entities/{idOrKey}/links/targets")
     @Timed
     @PostFilter("hasPermission({'returnObject': filterObject, 'log': false}, 'XMENTITY.LINK.TARGET.GET_LIST')")
+    @PrivilegeDescription("Privilege to get all the targets")
     public List<Link> getLinkTargets(@PathVariable String idOrKey, @RequestParam(required = false) String typeKey) {
         return xmEntityService.getLinkTargets(IdOrKey.of(idOrKey), typeKey);
     }
 
     @PostMapping("/xm-entities/{idOrKey}/links/targets")
     @Timed
     @PreAuthorize("hasPermission({'idOrKey':#idOrKey, 'link':#link, 'file':#file}, 'XMENTITY.LINK.TARGET.CREATE')")
+    @PrivilegeDescription("Privilege to save target")
```

Comment:
Add link into entity

Reply:
suggest:
`Privilege to add target link to indicated xmEntity`